src/lib/components/inbox/conversations/conversation_snippet.tsx

```diff
@@ -109,7 +111,7 @@ export class ConversationSnippet extends React.Component<Props, any> {
     const artworkDate = `${artwork.date}`
     const artworkArtist = `${artwork.artist.name} · `
     const conversationText = conversation.last_message.replace(/\n/g, " ")
-    const date = "11:00 AM"
+    const date = moment(conversation.last_message_at).fromNow(true)
```

Comment:
from what I gather it would be `2 hours ago`, but my comment was that from the designs it looks like it should be `11:00 AM`